### ISSUE TITLE
secrets: Add log messages

### ISSUE THREAD

**Issue by robin-nitrokey** [Nitrokey/nitrokey-app2#98]
## (1) sdk log messages- already works

`init_logging` calls `logging.basicConfig(level=DEBUG, handlers=[FileHandler])`, which configures the root logger, and no sdk logger sets `propagate=False` or installs a `NullHandler` (anything the sdk logs already ends up in the app2 log file)

for secrets it is already covered. `SecretsApp.__init__` takes an optional `logfn` and falls back to `self.log.info`, with `self.log = logging.getLogger("otpapp")`

app2 constructs `SecretsApp(device)` at all 16 call sites without passing `logfn`, so the fallback applies and the `self.logfn` calls in `_send_receive_inner` and others are live

note: `"otpapp"` is inconsistent with `__name__` used everywhere else in the sdk, so these lines cannot be filtered by the `nitrokey.` prefix (renaming to `nitrokey.nk3.secrets_app`)

the rest of the sdk logs very little, except the vendored lpc55_upload/nrf52_upload trees, 0.5.0rc2 has log calls only in:

- `trussed/updates.py` — 11 calls, firmware update path
- `trussed/_bootloader/*` — bootloader only
- `trussed/_device.py` — 2 warnings, only when opening a device fails
- `trussed/admin_app.py` — 2 debug, only after reboot

so on a normal session (enumerated devices, read status, fido2 operations) the sdk contributes nothing outside secrets. nothing in `admin_app.status/version/uuid`, nothing on the fido2 paths. covering (1) outside secrets has to happen in the sdk, there is nothing to wire up on the app2 side (I suggest splitting that into a separate issue on the sdk repo)

## (2) action logging- not there

every device action goes through `Worker.run(job)` (15 dispatch slots across `SecretsWorker`, `Fido2Worker`, `SettingsWorker`, `OverviewWorker`, and 18 `Job` subclasses) and nothing logs there. that single method is the chokepoint, so one log line covers all of it. 
`update.py` is the exception, it already logs button clicks.

the other `triggered.connect` handlers in the tabs are ui (clipboard copy, show password, toggle otp edit) and never touch the device (left them alone)

## (3) error logging- absent

adding it to `Job.trigger_error` and `Job.trigger_exception` works as suggested, with one catch: 
- two subclasses override `trigger_error` without calling super `VerifyPinJob` (`secrets_tab/worker.py:167`) and `SavePasswordsPinJob` (`settings_tab/worker.py:146`) both skip emitting `finished` on purpose, so delegating to super would change the behaviour (add a log line in each instead)

## unrelated bugs found in Job

- `trigger_error`/`trigger_exception` concatenate class name and message with no separator, so the info box shows `ListCredentialsJobPIN validation failed: ...`
- `trigger_exception` is decorated `@Slot(str, Exception)` but takes a single argument

## proposed split

- this issue: (2) and (3) in app2
- new issue on nitrokey-sdk-py: log calls in admin_app and the fido2/device paths, and renaming the `otpapp` logger
